src/redux/slice/workspaceSlice.ts

```diff
@@ -176,6 +194,23 @@ const workspacesSlice = createSlice({
                 state.loading = false
                 state.error = action.payload
             })
+            .addCase(makeAdmin.pending, state => {
+                state.loading = true
+                state.error = null
+            })
+            .addCase(makeAdmin.fulfilled, (state, action) => {
+                state.loading = false
```

Comment:
add  the error clean up as well
```suggestion
                state.loading = false
                state.error = null
```